### DISCUSSION TITLE
Endringer på Linkpanel

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#1890] [Komponenter]
🔗 [Link til Linkpanel i Aksel](https://aksel.nav.no/komponenter/core/linkpanel)

Vi ønsker å vurdere hva som kan forbedre brukeropplevelsen av Linkpanel for utviklere og designere. Har du innspill til noe som vil gjøre denne komponenten bedre og mer brukervennlig for deg og brukerne av deres løsninger?

**Comment by heyvard** [comment]
Jeg ønsker meg gjerne mulighet for å sette border og background color med tokens.
I tillegg gjerne at den indre divven har width 100%. Nå nå vi style den 100% manuelt i en del tilfeller.

**Comment by daghelgescott** [comment]
Enig med **heyvard** når det kommer til border og background color.  

**I tillegg ønsker jeg meg muligheten til å:**
- endre radius og padding, slik at store lenkepaneler kan få større radius og padding enn mindre lebkepaneler (evt. small-, medium-, large-versjon)
- legge til shadow tokens på default state (og endre tokens på hover). Gjelder spesielt panelene uten border.
- legge til [tags](https://aksel.nav.no/komponenter/core/tag). Disse kan kanskje legges nederst i panelet, sammen med chevron?
- apropos tags: det hadde vært fint med en versjon uten border, men med svakere bakgrunnsfarge enn den fylte versjonen. Nå blir det litt sånn boks-i-boks følelse når jeg bruker tags med border. 
- endre tekststørrelser i tittel og description.

Vet ikke om det er på dette nivået dere vil ha innspill, men nå har dere i hvert fall fått noen 🤗

**Comment by sjur-gr** [comment]
Gracias amigos! Dette er supre innspill 🤘

**Comment by karihamnes** [comment]
Mulig det er litt seint, men jeg ønsker meg small-variant av linkpanel for interne flater. Og mulighet for ikon, liknende som expansion card?
